lib/gpu/Makefile.hip

```diff
@@ -137,7 +137,7 @@ $(OBJ_DIR)/lal_%.o: lal_%.cpp $(CUHS) $(ALL_H)
 
 $(LIB_DIR)/libgpu.a: $(OBJS)
 	$(AR) -crs $@ $(OBJS)
-	echo "export HIP_PLATFORM := $(HIP_PLATFORM)\n$(HIP_LIBS_TARGET)" > 'Makefile.lammps'   
+	echo -e "export HIP_PLATFORM := $(HIP_PLATFORM)\n$(HIP_LIBS_TARGET)" > 'Makefile.lammps'   
```

Comment:
I recommend this be changed to printf, as `echo -e` is [not portable](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/echo.html#tag_20_37_16).  I noticed this as compiling fails on Ubuntu 16.04